script/column.js

```diff
@@ -0,0 +1,30 @@
+import createElement from './createElement.js';
+
+export default class Column {
+    constructor(heading) {
+        this.heading = heading;
+        this.container = null;
+        this.render();
+    }
+
+    get elem() {
+        return this.container;
+    }
+
+    columnTemplate(heading) {
+        return `
+        <div class="board-column">
+            <div class="board-column-header">
+                <h2>${heading}</h2>
+            </div>
+            <div class="board-column-body">
```

Comment:
It's good to have the column body as the DOM element too and keep it as a field of the Column class. So it'll be easier to append the ticket there later